src/main/java/com/impactupgrade/nucleus/controller/PaymentGatewayController.java

```diff
@@ -353,4 +355,34 @@ public Response verifyAndReplayDeposits(
 
     return Response.status(200).build();
   }
+
+  @GET
+  @Path("/sync/contacts/daily")
+  public Response syncDaily(@QueryParam("syncDays") Integer syncDays, @Context HttpServletRequest request) throws Exception {
+    Environment env = envFactory.init(request);
+
+    Calendar lastSync = Calendar.getInstance();
+    if (syncDays == null || syncDays <= 0) {
+      syncDays = 3;
+    }
+    lastSync.add(Calendar.DATE, -syncDays);
+
+    Runnable thread = () -> {
+      env.startJobLog(JobType.EVENT, null, "Contacts Sync: Daily", "Nucleus Portal");
+
+      SynchronizationService synchronizationService = env.synchronizationService("stripeSync");
```

Comment:
Ditto. Let's consolidate all this into a single Controller endpoint? A new Controller is fine if that makes more sense. Make it a single sync-contacts endpoint that loops over all SynchronizationServices?